examples/subscribe/main.go

```diff
@@ -8,12 +8,17 @@ import (
 	"github.com/davecgh/go-spew/spew"
 	"github.com/gozwave/gozw"
 	"github.com/gozwave/gozw/cc"
+	"github.com/gozwave/gozw/cc/allclasses"
 )
 
 var networkKey = []byte{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00}
 
+func init() {
+	allclasses.Noop()
+}
+
 func main() {
-	client, err := gozw.NewDefaultClient("/tmp/data.db", "/dev/tty.usbmodem1431", 115200, networkKey)
+	client, err := gozw.NewDefaultClient("/tmp/data.db", "/dev/tty.usbmodem1441", 115200, networkKey)
```

Comment:
It might be nice to have an envvar for this so we can keep our own configs

Reply:
Agreed. Coming right up!